forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePANEditText.kt

```diff
@@ -173,6 +175,11 @@ class ForagePANEditText @JvmOverloads constructor(
             }
     }
 
+    override fun showKeyboard() {
+        val imm = context.getSystemService<InputMethodManager>()
+        imm!!.showSoftInput(textInputEditText, 0)
+    }
```

Comment:
Back to my other comment, I think there is a limitation in Android against your proposal. Should be confirmed though.

Reply:
So `requestFocus` is a method that our `Forage*Element`s inherent from `View`. Currently there are three method signatures for `requestFocus` that our `Forage*Element`s already have:
1. `requestFocus(int direction, Rect previouslyFocusedRect)`
2. `requestFocus(int direction)`
3. `requestFocus()`

Because method #3 already exists, we can't declare our own `requestFocus(showKeyboard: Boolean = false)` because the compiler has no way to disambiguate between whether `requestFocus()` is the OG `requestFocus()` or the Forage `requestFocus()` (default `false`).